examples/express/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "splunk-otel-example-express",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Example of instrumenting a toy Express app in different ways",
+  "main": "server.js",
+  "scripts": {
+    "client": "env-cmd nodemon -r ./tracer.js client.js",
```

Comment:
So this `-r` is smart, but let's highlight it in the README. There's just one `server.js` file and it's not immediately obvious how it's instrumented.

Reply:
Rewrote the overview section.